docs/medical-api/more-info/webhooks.mdx

```diff
@@ -359,3 +359,45 @@ Example payload:
   ]
 }
 ```
+
+### Bulk Document Download URLs
+
+While you can download indiviudal specified documents using the [get document url endpoint](/medical-api/api-reference/document/get-document), 
+patients might have a large number of documents. 
+
+If you want to download all of a patient's documents, especially when they have a large volume of documents, you can instead start a [Bulk Get Document URL query](/medical-api/api-reference/document/bulk-get-document-url) - we’ll send a webhook message with all the downloadable URLs for all the patients documents.
```

Comment:
nit: this was referred to as the "Bulk URL Generation endpoint" previously, but "Bulk Get Document URL" here

perhaps let's refer to it as "Bulk Get Document URL" in both places for consistency?